dlp/snippets/jobs/jobs_test.go

```diff
@@ -135,7 +135,7 @@ func riskNumerical(projectID, dataProject, pubSubTopic, pubSubSub, datasetID, ta
 		cancel()
 	})
 	if err != nil {
-		return fmt.Errorf("Recieve: %w", err)
+		return fmt.Errorf("Recieve: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.